packages/application-config/src/load-config.ts

```diff
@@ -0,0 +1,32 @@
+import type { JSONSchemaForCustomApplicationConfigurationFiles } from './schema';
+
+import { cosmiconfigSync, defaultLoaders } from 'cosmiconfig';
+
+const moduleName = 'applicationconfig';
```

Comment:
With the naming of this file. I wonder if `applicationconfig` is a bit too "wide". Maybe `merchantCenterCustomApplicationConfig`? Dang it that's long or `customApplicationConfig? Not 100% sure either.

Reply:
Yeah I though about that but the full name felt too long. Maybe `customapplicationconfig` if you prefer so.

Reply:
Yeah. I would think having some "branding" can help to users not confuse things. E.g. `customapplicationconfig` or `custom-application-config`?

Reply:
Renamed to `custom-application-config` 👌